code/modules/projectiles/guns/manufacturer/clip_lanchester/ballistics.dm

```diff
@@ -398,7 +398,7 @@ NO_MAG_GUN_HELPER(automatic/smg/cm5)
 
 /obj/item/gun/ballistic/automatic/marksman/f90
 	name = "CM-F90"
-	desc = "A powerful sniper rifle used by vanishingly rare CLIP specialists, capable of impressive range and penetrating power. Chambered in 6.5mm CLIP."
+	desc = "A powerful sniper rifle used by vanishingly rare CLIP specialists, featuring the impressive range and penetrating power characteristic of a sniper rifle. Chambered in 6.5mm CLIP."
```

Comment:
Second change is pretty redundant since it was already described as a sniper rifle right at the start. I'd suggest...

"A powerful sniper rifle used by vanishingly rare CLIP specialists, featuring impressive range and penetrating power on a relatively lightweight platform. Chambered in 6.5mm CLIP."